fintech-ui/src/app/app-routing.module.ts

```diff
@@ -14,6 +14,7 @@ const routes: Routes = [
   },
   {
     path: 'search',
+    //   canActivate: [AuthGuard],
```

Comment:
What's the reason to uncomment this line?

Reply:
because we haven't actually implemented the login logic and I think it will be better to comment it after login in done